index.html

```diff
@@ -24,7 +24,14 @@ <h1>Welcome to the student showcase!</h1>
         <div>FOUR</div>
         <div>FIVE</div>
         <div>SIX</div>
-        <div>SEVEN</div>
+        <div>
+          <h1>Jennifer Fisher</h1>
+          <<img src="https://avatars0.githubusercontent.com/u/38234245?s=460&v=4" width="150" controls/>
```

Comment:
`width` is an inline style. Don't use those (save them for CSS). And you also shouldn't need the `controls` attribute on an `img`.